portfolio/src/test/java/com/google/sps/DeleteCommentsTest.java

```diff
@@ -0,0 +1,129 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
```

Comment:
the package should generally match the file path for the java classes. so this file needs to be moved into `portfolio/src/test/java/com/google/sps/servlets` directory.

Reply:
ah ok! thank you!

Reply:
my comment had a typo. it's now fixed. The correct path is `portfolio/src/test/javatests/com/google/sps/servlets`.

Reply:
One more correction: `portfolio/src/test/java/com/google/sps/servlets`